### PR TITLE
Add versioned true to exp tracking README

### PR DESCRIPTION
**Pull request by limdauto**
## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +104,34 @@ Options:
...
+SESSION_STORE_CLASS = SQLiteStore
+SESSION_STORE_ARGS = {"path": str(Path(__file__).parents[2] / "data")}
+```
+
+Once the above set-up is complete, tracking datasets can be used to track relevant data for Kedro runs. More information on how to use tracking datasets can be found [here](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html)
+
+**Notes:**
+
+* Experiment Tracking is only available for Kedro-Viz >= 4.0.2 and Kedro >= 0.17.5
+* Prior to Kedro 0.17.6, when using `tracking.MetricsDataSet`, you will have to explicitly marked the dataset as `versioned` for it to show up properly in Kedro-Viz experiment tracking tab. From Kedro >= 0.17.6, this is done automatically:
```
Comment (studioswong) [inline README.md]: Does this apply to all tracking datasets? I remember in setting up modular-spaceflights I have to apply this to `tracking.JSONDataSet` as well.

File: README.md [hunk 2]
```diff
@@ -120,7 +104,34 @@ Options:
   -h, --help                Show this message and exit.
 ```
 
-### As a standalone React component
+### Experiment Tracking usage
+
+To enable [experiment tracking](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html) in Kedro-Viz, you need to add the Kedro-Viz SQLiteStore to your Kedro project.
```
Comment (MerelTheisenQB) [inline README.md:109]:
```suggestion
To enable [experiment tracking](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html) in Kedro-Viz, you need to add the Kedro-Viz `SQLiteStore` to your Kedro project.
```

**Review by MerelTheisenQB — APPROVED** [review]
LGTM. Just to double check: we have tested that this actually works? During the hackathon we did slightly modify the dataset implementation as well to make this work.

**Comment by limdauto** [comment]
@MerelTheisenQB Yes, tested and working.

File: README.md (hunk 1 above)
Comment (limdauto) [reply README.md]: Oh good catch! Sorry, all tracking datasets.

**Pull request merged**